src/__stories__/button-layout-story.tsx

```diff
@@ -41,58 +41,58 @@ export const Default: StoryComponent = () => (
         <div data-testid="screenshot">
             <StorySection title="ButtonLayout">
                 <ButtonLayout>
-                    <ButtonSecondary fake>ok</ButtonSecondary>
-                    <ButtonPrimary fake>long long long</ButtonPrimary>
+                    <ButtonSecondary fake>Ok</ButtonSecondary>
+                    <ButtonPrimary fake>The text in this button is very long</ButtonPrimary>
```

Comment:
In the mobile screenshots of this story I miss a case where both buttons are side by side

Reply:
ok